www/docs/en/dev/guide/platforms/windows/plugin.md

```diff
@@ -66,11 +68,26 @@ cordova.commandProxy.add("Echo",{
         }
     }
 });
+
+// or alternative syntax
```

Comment:
No idea if this actually works - will have to try it and compare that to the currently documented implementation.

Reply:
Those are probably both actually calling https://github.com/apache/cordova-js/blob/a770d95f129f0e16c696bcecd00dc8d3e8e53b5c/src/common/exec/proxy.js